content/Products/OpenshiftMonitoring/instrumentation.md

```diff
@@ -0,0 +1,80 @@
+# Instrumentation guidelines
+
+This document details good practices to adopt when you instrument your application for Prometheus. It is not meant to be a replacement of the [upstream documentation](https://prometheus.io/docs/practices/instrumentation/) but an introduction focused on the OpenShift use case.
+
+## Targeted audience
+
+This document is intended for OpenShift developers that want to instrument their operators and operands for Prometheus.
+
+## Getting started
+
+To instrument software written in Golang, see the official [Golang client](https://pkg.go.dev/github.com/prometheus/client_golang). For other languages, refer to the [curated list](https://prometheus.io/docs/instrumenting/clientlibs/#client-libraries) of client libraries.
+
+Prometheus stores all data as time series which are a stream of timestamped values (samples) identified by a metric name and a set of unique labels (a.ka. dimensions or key/value pairs). Its data model is described in details in this [page](https://prometheus.io/docs/concepts/data_model/). Time series would be represented like this:
+
+```
+http_requests_total{method="GET", handler="/messages"}  500
+http_requests_total{method="POST", handler="/messages"} 10
```

Comment:
I know from a data model perspective this is correct. In reality user will also see 
```
# HELP
# TYPE
```
Do you think it would be too detailed to mention that in this doc?

Reply:
Nope it makes sense. I'll add it.